aten/src/ATen/native/cpu/ReduceOpsKernel.cpp

```diff
@@ -58,7 +58,7 @@ static void norm_kernel_tensor_iterator_impl(
     TensorIterator& iter,
     Scalar p) {
   float val;
-  if (p.isIntegral()) {
+  if (p.isIntegral(true)) {
```

Comment:
any particular reason for this? It seems slightly nicer to just make this an error and we can turn it on if people ask.